client/cmdflashmem.c

```diff
@@ -15,9 +15,10 @@
 static int CmdHelp(const char *Cmd);
 int usage_flashmem_read(void){
 	PrintAndLogEx(NORMAL, "Read flash memory on device");
-	PrintAndLogEx(NORMAL, "Usage:  mem read o <offset> l <len>");
+	PrintAndLogEx(NORMAL, "Usage:  mem read o <offset> l <len> [f]");
 	PrintAndLogEx(NORMAL, "  o <offset>    :      offset in memory");
 	PrintAndLogEx(NORMAL, "  l <len>       :      length");
+	PrintAndLogEx(NORMAL, "  f             :      fastRead mode");
```

Comment:
the mix  of param usage is annoying.    F  for fastread here,  and + for it later...

Reply:
Yes I first used f then it was already taken for save... let say F sensitive case, would fit ?

Reply:
I dislike F / f having different meanings.   I do like the '+' idea,  but   why not  'q'  as in QuickRead?

Reply:
Ok better idea: since write operation and status register reading are virtually unaffected by the switch from 24MHz to 48MHz, why not a switchable state (like for mfg_dbg) which allows choosing the SCBR ? 1=48MHz, others are divisors, and if we use >=33Mhz device shouldn't use fastread but read instead, and voila. ?

Reply:
I like that solution even better